### ISSUE TITLE
BUG: Search field not expanding to 100% width of the region on CLAS Policy site

### ISSUE THREAD

**Issue by mark-bennett-uiowa** [uiowa/uiowa#5809]
The search field on the front page of the CLAS Policy and Procedure site https://policy.clas.uiowa.edu does not expand to fill the entire region. It appears that the element below needs a width:100%, but there might be something else causing this.

`<div class="js-form-item form-item js-form-type-search-api-autocomplete form-type-search-api-autocomplete js-form-item-text form-item-text form-no-label" style="width: 100%;">...</div>`

<img width="1037" alt="Screen Shot 2022-10-04 at 10 02 24 AM" src="https://user-images.githubusercontent.com/3067017/193855019-8e33fe34-faad-4506-aede-332eb97f8403.png">

**Comment by bspeare** [comment]
@mark-bennett-uiowa what do you think about switching to the Search Block for this functionality?  I think this is the only "custom" search block on the platform that was built before we released the "Search Block" for v3.

**Comment by mark-bennett-uiowa** [comment]
This was due to the site using a custom predictive text search field. After discussion with @bspeare we switched this over to the system's Search widget, and added the proper parameter to achieve the same results, and fix the width issue of the search field. Though this new field does not have the same predictive text functionality, that will not be a missed feature, and we can close this issue as Fixed.
